ratis-server/src/main/java/org/apache/ratis/server/storage/SnapshotManager.java

```diff
@@ -116,8 +116,6 @@ public void installSnapshot(StateMachine stateMachine,
           LOG.warn("The snapshot md5 digest {} does not match expected {}",
               digest, expectedDigest);
           // rename the temp snapshot file to .corrupt
-//          NativeIO.renameTo(tmpSnapshotFile, // TODO:
-//              dir.getCorruptSnapshotFile(lastIncludedTerm, lastIncludedIndex));
```

Comment:
Hi @dineshchitlangia, why is this rename no longer required?

Reply:
@arp7 good catch. These lines were commented out and since we are removing the entire native library, I removed the commented line of code.
When I look at this piece again, I think that we can remove these NativeIO line of code from the comment but add a TODO note regarding rename.
Would that be fine ?